oak_tensorflow_service/src/tflite.rs

```diff
@@ -66,51 +70,55 @@ const TENSOR_ARENA_SIZE: usize = 1024;
 #[derive(Default)]
 pub struct TfliteModel {
     tensor_arena: Vec<u8>,
+    output_buffer_len: Option<usize>,
 }
 
 impl TfliteModel {
     pub fn new() -> Self {
         let tensor_arena = vec![0; TENSOR_ARENA_SIZE];
-        Self { tensor_arena }
+        Self {
+            tensor_arena,
+            output_buffer_len: None,
+        }
     }
 
-    pub fn initialize(&self, model_bytes: &[u8]) -> anyhow::Result<()> {
-        let model_bytes_len = model_bytes
-            .len()
-            .try_into()
-            .map_err(|error| anyhow!("Failed to convert usize to u64: {}", error))?;
-        let tensor_arena_len = self
-            .tensor_arena
-            .len()
-            .try_into()
-            .map_err(|error| anyhow!("Failed to convert usize to u64: {}", error))?;
+    pub fn initialize(&mut self, model_bytes: &[u8]) -> anyhow::Result<()> {
+        let model_bytes_len = model_bytes.len();
+        let tensor_arena_len = self.tensor_arena.len();
+        let mut output_buffer_len = 0usize;
         let _ = unsafe {
             tflite_init(
                 model_bytes.as_ptr(),
                 model_bytes_len,
                 self.tensor_arena.as_ptr(),
                 tensor_arena_len,
+                &mut output_buffer_len,
             )
         };
+        self.output_buffer_len = Some(output_buffer_len);
```

Comment:
Should we log something here? It may be useful for debugging

Reply:
Done